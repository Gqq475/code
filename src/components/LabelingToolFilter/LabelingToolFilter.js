import React from 'react'
import classes from './LabelingToolFilter.scss'

type Props = {

};
export class LabelingToolFilter extends React.Component {
  props: Props;

  render () {
    return (
      <div className={classes['LabelingToolFilter-container']} />
    )
  }
}

export default LabelingToolFilter
