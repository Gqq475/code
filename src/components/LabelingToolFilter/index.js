import LabelingToolFilter from './LabelingToolFilter'
export default LabelingToolFilter
