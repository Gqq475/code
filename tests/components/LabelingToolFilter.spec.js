import React from 'react'
import LabelingToolFilter from 'components/LabelingToolFilter/LabelingToolFilter'

describe('(Component) LabelingToolFilter', () => {
  it('render normally', () => {
    const wrapper = shallow(
      <LabelingToolFilter />
    )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
