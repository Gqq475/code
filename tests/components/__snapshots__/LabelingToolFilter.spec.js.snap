// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) LabelingToolFilter render normally 1`] = `
<div
  className="LabelingToolFilter-container"
/>
`;
